docs/rpc/api/core-node/get-burn-ops.example.json

```diff
@@ -1,14 +1,41 @@
 {
-  "peg_in": [
-    {
-      "amount": 1337,
-      "block_height": 218,
-      "burn_header_hash": "3292a7d2a7e941499b5c0dcff2a5656c159010718450948a60c2be9e1c221dc4",
-      "memo": "",
-      "peg_wallet_address": "tb1pqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqkgkkf5",
-      "recipient": "S0000000000000000000002AA028H.awesome_contract",
-      "txid": "d81bec73a0ea0bdcf9bc011f567944eb1eae5889bf002bf7ae641d7096157771",
-      "vtxindex": 2
-    }
-  ]
+    "peg_in": [
+        {
+            "amount": 1337,
+            "block_height": 218,
+            "burn_header_hash": "3292a7d2a7e941499b5c0dcff2a5656c159010718450948a60c2be9e1c221dc4",
+            "memo": "",
+            "peg_wallet_address": "tb1pqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqkgkkf5",
+            "recipient": "S0000000000000000000002AA028H.awesome_contract",
+            "txid": "d81bec73a0ea0bdcf9bc011f567944eb1eae5889bf002bf7ae641d7096157771",
+            "vtxindex": 2
+        }
+    ],
+    "peg_out_request": [
+        {
+            "amount": 1337,
+            "recipient": "1BixGeiRyKT7NTkJAHpWuP197KXUNqhCU9",
+            "block_height": 218,
+            "burn_header_hash": "3292a7d2a7e941499b5c0dcff2a5656c159010718450948a60c2be9e1c221dc4",
+            "peg_wallet_address": "tb1qqvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvps3f3cyq",
+            "txid": "d81bec73a0ea0bdcf9bc011f567944eb1eae5889bf002bf7ae641d7096157771",
+            "vtxindex": 2,
+            "signature": "0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d",
+            "fulfillment_fee": 0,
+            "memo": "00010203"
+        }
+    ],
+    "peg_out_fulfill": [
```

Comment:
Is this example actually a possible return value for the RPC? I don't think it can contain `peg_out_fulfill` at the same time as `peg_out_request`. The OpenAPI spec can contain multiple examples, but I think they'd need to be in separate example files.

Reply:
I think we wanted to show all variations of the response. It seems there's a better way - add multiple examples to the spec.

https://www.viralpatel.net/openapi-multiple-examples/

Reply:
Thanks for highlighting this! Let's update to use separate examples to the spec then. @stjepangolemac will you do it or should I?

Reply:
I've updated it now :heavy_check_mark:

Reply:
Yeah I was trying to push but thought my git broke down. I didn't realize you pushed identical changes first so that confused me. Thanks!